src/notation/qml/MuseScore/NotationScene/internal/PercussionPanelPad.qml

```diff
@@ -41,6 +41,13 @@ DropArea {
     property alias showOriginBackground: originBackground.visible
     property alias draggableArea: draggableArea
 
+    property int navigationRow: -1
+    property int navigationColumn: -1
+    property alias padNavigationCtrl: padNavCtrl
```

Comment:
It's better to provide a panel property, it looks more logical for this file

Reply:
I decided on this approach because there are actually two panels (one for the pads and one for the footers) which have the same navigation row/column.

Reply:
two options: give two panels or remove Ctrl from the name (like for example for FlatButton)
I don't like Ctrl because it clearly shows that we are giving the private part out